admin/appendix.rst

```diff
@@ -2450,25 +2920,96 @@ This tool can be used to obfuscate passwords stored in server and agent
 configuration files as well as various places in the system, e.g. ssh passwords,
 notification channel passwords, etc.
 
+Usage:
+       nxencpasswd [<options>] <login> [<password>]
+       nxencpasswd [<options>] -a [<password>]
+
+Options:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+   
+   * - Source
+     - Description
+   * - -a
+     - Encrypt agent's secret.
+   * - -h
+     - Display help and exit.
+   * - -v
+     - Display version and exit.
+
+.. note::
+
+     If password is not provided it will be requested from terminal.    
+
+
 nxevent
--------
+--------
+
+Nxevent is installed with NetXMS client distribution. Sends events to server directly.
+
+Usage: nxevent [<options>] <server> <event> [<param_1> [... <param_N>]]
+
+Options:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+   
+   * - Source
+     - Description
+   * - -c
+     - Codepage (default is ISO8859-1).
+   * - -C <count>
+     - Repeat event sending given number of times.
+   * - -d
+     - Turn on debug mode.
+   * - -e
+     - Encrypt session (for compatibility only, session is always encrypted).
+   * - -h
+     - Display help and exit.
+   * - -i <interval>
+     - Repeat event sending with given interval in milliseconds.
+   * - -n
+     - Parameters are provided in named format (name=value).
+   * - -o <id>
+     - Specify source object ID.
+   * - -P <password>
+     - Specify user's password. Default is empty password.
+   * - -S
+     - Skip protocol version check (use with care).
+   * - -T <tag>
+     - User tag to be associated with the message. Default is empty.
+   * - -u <user>
+     - Login to server as <user>. Default is "guest".
+   * - -v
+     - Display version and exit.
+   * - -w <seconds> 
+     - Specify command timeout (default is 3 seconds).
+
+
+Example
+
+
+
+Send event to server:
+
+  .. code-block:: sh
+
+     nxevent -u admin -P adminpassword localhost SYS_NODE_DOWN
 
-This tool can be used to push events to |product_name| server.
 
 nxget
 -----
 
 This tool is intended to get values of :term:`Metric` from |product_name| agent.
 
-Syntax:
-
-.. code-block:: shell
-
-   nxget [options] host [metric [metric ...]]
+Usage: nxget <host> [<options>] [<parameter> [<parameter> ...]]
 
 Where *host* is the name or IP address of the host running |product_name| agent; and
-*metric* is a metric, list or table name, depending on given options. By default,
-nxget will attempt to retrieve the value of only one given metric, unless *-b* option is given.
+*parameter* is a metric, list or table name, depending on given options. By default,
+nxget will attempt to retrieve the value of only one given parameter, unless *-b* option is given.
 
```

Comment:
When metric is requested without explicitly specifying metric type (table or list), nxget attempts to get values trying types in the following order: singe-value metric, table, list.